src/components/App/App.jsx

```diff
@@ -1,9 +1,38 @@
-import ProjectForm from "../ProjectForm/ProjectForm";
-
+import { Home } from "./Home.jsx";
+import NavBar from "./NavBar";
+import { Typography } from "@material-ui/core";
+import { Route, BrowserRouter as Router, Switch } from "react-router-dom";
 import "./App.scss";
 
 function App() {
-  return <ProjectForm />;
+  return (
+    <>
+      <Router>
+        <NavBar></NavBar>
+        <div className="content">
+          <Switch>
+            <Route component={Home} exact path="/"></Route>
+            <Route exact path="/projects">
+              <Typography
+                style={{ color: "#9E9EAA", fontWeight: "700" }}
+                variant="h4"
+              >
+                Projects coming soon!
+              </Typography>
+            </Route>
+            <Route exact path="/resumes">
+              <Typography
+                style={{ color: "#9E9EAA", fontWeight: "700" }}
+                variant="h4"
+              >
+                Resumes coming soon!
+              </Typography>
+            </Route>
+          </Switch>
+        </div>
+      </Router>
+    </>
```

Comment:
Just one small thing, fragment here is unnecessary because you can use the Route component as parent element here.
```
return (
      <Router>
        <NavBar/> // note how a component can be closed 
        ...
      </Router>
)
```